### DISCUSSION TITLE
Disable ligatures

### DISCUSSION THREAD

**Discussion by sigurdo** [sourcelair/xterm.js#4854] [Q&A] [answered]
Font ligatures seem to be enabled by default, but I don't want it, so how can I disable it?

**Comment by Tyriar** [comment]
They're not enabled by default, unless you are using the `xterm-addon-ligatures` module

**Comment by sigurdo** [comment]
No, for me they are enabled by default without the addon, see this example for reference: https://github.com/sigurdo/demo-xterm-js-no-ligatures.

**Reply by Tyriar** [reply, marked as answer]
Hmm, I'm not sure then. I can't reproduce on the regular demo. Since you're using the standard demo though, you could force ligatures to be disabled in CSS with:

```
font-variant-ligatures: none !important;
font-feature-settings: "liga" 0 !important;
```

**Reply by sigurdo** [reply]
Thanks, that does the job. But would you mind elaborating on what you mean by regular and standard demo, just since I'm curious? I just followed the steps in the README and put together my own demo.

**Reply by Tyriar** [reply]
This is the file that's run in the xterm.js demo https://github.com/xtermjs/xterm.js/blob/master/demo/client.ts, see https://github.com/xtermjs/xterm.js/wiki/Contributing#running-the-demo
